src/libs/API/parameters/UpdatePolicyConnectionConfigParams.ts

```diff
@@ -4,7 +4,7 @@ type UpdatePolicyConnectionConfigParams<TConnectionName extends ConnectionName,
     policyID: string;
     connectionName: TConnectionName;
```

Comment:
Remove generic types. No longer useful

Reply:
Yes but do not change the `updatePolicyConnectionConfig` function. The parameters there should be still typed and use `UpdatePolicyConnectionConfigParams`

Reply:
This change will affect setting name.

Reply:
Yes that's what I meant, earlier I was on phone and I didn't know how to select multiple lines to comment on 😅 Is there a problem in changing the settings name?

Reply:
Actually at this point this does not matter much. Now we are sending the correct data to the server. The ts can be polished later

Reply:
when you remove the generic type then we have a situation for setting name. check screenshots
<img width="720" alt="Screenshot 2024-04-22 at 15 37 53" src="https://github.com/Expensify/App/assets/22779249/f20d6916-01e6-4815-a046-d46effe6b8b2">
<img width="849" alt="Screenshot 2024-04-22 at 15 38 12" src="https://github.com/Expensify/App/assets/22779249/4acc0a34-1e4d-40c9-b487-2852b7c51310">

Reply:
Same with.idemptonecyKey. Convert to String

Reply:
I can set the `setting name ` in `updatePolicyConnectionConfig` to string. if that works for you.